VideoWeb/VideoWeb/ClientApp/src/app/services/conference/video-control-cache.service.spec.ts

```diff
@@ -0,0 +1,241 @@
+import { TestBed } from '@angular/core/testing';
+import { LoggerService } from '../logging/logger.service';
+
+import { IHearingControlsState, IHearingControlStates, VideoControlCacheService } from './video-control-cache.service';
+
+describe('VideoControlCacheService', () => {
+    let sut: VideoControlCacheService;
+    let loggerSpy: jasmine.SpyObj<LoggerService>;
+
+    beforeEach(() => {
+        loggerSpy = jasmine.createSpyObj<LoggerService>('LoggerService', ['warn', 'info']);
+        sut = new VideoControlCacheService(loggerSpy);
```

Comment:
I know it's only got one dependency here but could we use TestBed for injecting the dependencies?

Reply:
@FarronF If we use testbed, would the karma test runner run this spec file slower?

Reply:
I don't think there is a benefit to using test bed for service tests is there? I thought it was just for components if you want to test the UI?

Reply:
Hmmm I guess I'm just used to using TestBed as the CLI provides it when you create a service. But after reading this https://www.rallyhealth.com/coding/testing-angular-services-skip-the-testbed

I'll have to read up on it some more but yeah seems fine.